src/pages/card/components/Tag.tsx

```diff
@@ -0,0 +1,160 @@
+import { Card } from "antd";
+import {
+  formatDate,
+  getCardStatusAndText,
+  hasAudios,
+  hasImages,
+  hasVideos,
+  UserRoles,
+} from "../../../utils/Extensions";
+import { CardInterface, Evidences } from "../../../data/card/card";
+import Strings from "../../../utils/localizations/Strings";
+import CustomTagV2 from "../../../components/CustomTagV2";
+import { useNavigate } from "react-router-dom";
+import { useMemo } from "react";
+import { AiOutlinePicture } from "react-icons/ai";
+import { IoHeadsetOutline } from "react-icons/io5";
+import { GoDeviceCameraVideo } from "react-icons/go";
+import {
+  adminCardDetails,
+  localAdminCardDetails,
+  sysAdminCardDetails,
+} from "../../routes/Routes";
+
+interface CardProps {
+  data: CardInterface;
+  rol: UserRoles;
+}
+
+const InformationPanel = ({ data, rol }: CardProps) => {
```

Comment:
should we rename this? i mean this is Tag, why we use informationpanel?

Reply:
Yes you are right. I made the change now the const is "Tag" Is that okay?

Reply:
Yes

Reply:
Change alredy madre